src/test/java/picard/illumina/IlluminaBasecallsToSamTest.java

```diff
@@ -316,11 +329,13 @@ public void testCorruptDataReturnCode() throws Exception {
      */
     private void runStandardTest(final int lane, final String jobName, final String libraryParamsFile,
                                  final int concatNColumnFields, final String readStructure,
-                                 final File baseCallsDir, final File testDataDir) throws Exception {
+                                 final File baseCallsDir, final File testDataDir, final Integer tile) throws Exception {
         final File outputDir = File.createTempFile(jobName, ".dir");
```

Comment:
Would be nice to replace this odd code with the Files API for creating a temp directory.